### PR TITLE
workflows: automate OpenSCAP config generation.

### PR DESCRIPTION
**Pull request by kingsleyzissou**
Add a workflow dispatcher for generating OpenSCAP configs. Also, create a schedule for the configs
to be checked once a week. If there are any changes create a pull request for this.

### CODE REVIEW

File: .github/workflows/openscap.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          branch: update-openscap
+          delete-branch: true
+          title: "distributions: regenerate OpenSCAP configs"
+          commit-message: "distributions: regenerate OpenSCAP configs"
+          body: Update OpenSCAP configs
+          token: ${{ secrets.###TODO### }}
```
Comment (kingsleyzissou) [inline .github/workflows/openscap.yml:50]: Would just need help setting this up

Comment (ochosi) [inline .github/workflows/openscap.yml:50]:
```suggestion
          token: ${{ secrets.SCHUTZBOT_GITHUB_ACCESS_TOKEN }}
```

I have shared this org-secret with this repo now, so you should be able to use it here.

**Review by ochosi — COMMENTED** [review]
Yay, thanks for looking into this! <3
Have you tested this somehow? In any case, the code does look very plausible :)
(Just to be clear, I have not.)

The secret I suggested should work for what you're trying to do, but I guess we can only find out once this is merged (unless you create the PR from a branch in the original repository and not your fork, which otherwise prevents secrets from being used etc.)

**Comment by kingsleyzissou** [comment]
> Have you tested this somehow?

I've tested as much as I can locally. The checkout and go stuff doesn't work great locally inside containers.
And using `OpenSCAP` directly on ubuntu is challenging. So I might have to tweak this a bit down the line.

File: .github/workflows/openscap.yml (hunk 1 above)
Comment (kingsleyzissou) [reply .github/workflows/openscap.yml:50]: Amazing. Thanks

**Comment by kingsleyzissou** [comment]
Whoops. I changed the base branch to `oscap-automation` and was going to create a new PR. In doing that, this PR got automatically merged to the new `oscap-automation` branch. The new PR is: https://github.com/osbuild/image-builder/pull/1160

**Pull request merged**